frontend/config/environment.js

```diff
@@ -21,6 +21,21 @@ module.exports = function(environment) {
       usingCors: false
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    "ember-simple-auth-token": {
+      serverTokenEndpoint: "/api/v1/auth/login",
+      identificationField: "username",
+      passwordField: "password",
+      tokenPropertyName: "token",
+      refreshTokenPropertyName: "token",
+      authorizationPrefix: "Bearer ",
+      authorizationHeaderName: "Authorization",
+      headers: {},
+      refreshAccessTokens: true,
+      serverTokenRefreshEndpoint: "/api/v1/auth/refresh",
+      tokenExpireName: "exp",
+      refreshLeeway: 300
```

Comment:
You are using all of the possible options but you are using a lot of default values. Remove options which are not custom